zio-http/src/main/scala/zio/http/ClientSSLConfig.scala

```diff
@@ -42,9 +52,22 @@ object ClientSSLConfig {
     )
   }
 
-  case object Default                                                                         extends ClientSSLConfig
-  final case class FromCertFile(certPath: String)                                             extends ClientSSLConfig
-  final case class FromCertResource(certPath: String)                                         extends ClientSSLConfig
-  final case class FromTrustStoreResource(trustStorePath: String, trustStorePassword: String) extends ClientSSLConfig
-  final case class FromTrustStoreFile(trustStorePath: String, trustStorePassword: String)     extends ClientSSLConfig
+  sealed trait ClientAuth
+  case object RequireClientAuth  extends ClientAuth
```

Comment:
@rajcspsg Can you put these in the companion object of `ClientAuth`?

Reply:
Also, once you put them there, they can have shorter names, e.g.:

```scala
object ClientAuth {
  case object Required extends ClientAuth
  case object None extends ClientAuth
  case object Optional extends ClientAuth
}
```

Reply:
@jdegoes sure. I've to set this config on ServerSslContext not on ClientSSLContext

Reply:
I'll update my PR shortly